components/hooks/useGetEvents.ts

```
@@ -0,0 +1,97 @@
+import { captureException } from "@sentry/nextjs";
+import useSWR, { preload } from "swr";
+import { PagedResponseDTO, EventSummaryResponseDTO } from "types/api/event";
+import { UseGetEventsProps } from "types/common";
+import { fetchEvents } from "../../lib/api/events";
+
+const fetcher = async (
+  _: any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)